File: src/main/java/com/demigodsrpg/chitchat/command/CCReloadCommand.java
package com.demigodsrpg.chitchat.command;

import com.demigodsrpg.chitchat.Chitchat;
import net.kyori.adventure.text.Component;
import net.kyori.adventure.text.format.NamedTextColor;
import org.bukkit.Bukkit;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class CCReloadCommand implements CommandExecutor {
    private final Chitchat INST;

    public CCReloadCommand(Chitchat inst) {
        INST = inst;
    }

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (sender.hasPermission("chitchat.reload")) {
            Bukkit.getServer().getPluginManager().disablePlugin(INST);
            Bukkit.getServer().getPluginManager().enablePlugin(INST);
        } else {
            sender.sendMessage(Component.text("You don't have permission to use that command.", NamedTextColor.RED));
        }
        return true;
    }
}

File: src/main/kotlin/com/demigodsrpg/chitchat/command/CCReloadCommand.kt
package com.demigodsrpg.chitchat.command

import com.demigodsrpg.chitchat.Chitchat
import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class CCReloadCommand(private val INST: Chitchat) : CommandExecutor {

    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (sender.hasPermission("chitchat.reload")) {
            Bukkit.getServer().pluginManager.disablePlugin(INST)
            Bukkit.getServer().pluginManager.enablePlugin(INST)
        } else {
            sender.sendMessage(ChatColor.RED.toString() + "You don't have permission to use that command.")
        }
        return true
    }
}